dlf/ext_tables.php

```diff
@@ -221,7 +221,9 @@
 t3lib_extMgm::addStaticFile($_EXTKEY, 'typoscript/', 'Basic Configuration');
 
 // Register plugins.
-t3lib_div::loadTCA('tt_content');
+if (version_compare(TYPO3_branch, '6.1', '<')) {
```

Comment:
You are right! Those lines are obsolete now.

Would you mind making a pull request with your proposed changes? (I think there are more obsolete version checks for PHP and TYPO3 version in the code.)